From e373f825ce1d538b04da2ad2cec53908f806dc4c Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Thu, 24 Oct 2013 17:42:47 +0800
Subject: [PATCH 257/602] ENGR00242365 Update BUILD_ID for jb4.3 ga rc2

Update BUILD_ID as "1.0.0-rc2"
Update BUILD_NUMBER as "20131108" to align new release date

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 arm2_6dq/build_id.mk       | 4 ++--
 arm2_6sl/build_id.mk       | 4 ++--
 evk_6sl/build_id.mk        | 4 ++--
 hdmidongle_6dq/build_id.mk | 4 ++--
 imx5x/build_id.mk          | 4 ++--
 sabreauto_6q/build_id.mk   | 4 ++--
 sabresd_6dq/build_id.mk    | 4 ++--
 7 files changed, 14 insertions(+), 14 deletions(-)

diff --git a/arm2_6dq/build_id.mk b/arm2_6dq/build_id.mk
index f850e0e..fe2c67b 100644
--- a/arm2_6dq/build_id.mk
+++ b/arm2_6dq/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-rc1
-export BUILD_NUMBER=20131029
+export BUILD_ID=1.0.0-rc2
+export BUILD_NUMBER=20131108
diff --git a/arm2_6sl/build_id.mk b/arm2_6sl/build_id.mk
index f850e0e..fe2c67b 100644
--- a/arm2_6sl/build_id.mk
+++ b/arm2_6sl/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-rc1
-export BUILD_NUMBER=20131029
+export BUILD_ID=1.0.0-rc2
+export BUILD_NUMBER=20131108
diff --git a/evk_6sl/build_id.mk b/evk_6sl/build_id.mk
index f850e0e..fe2c67b 100755
--- a/evk_6sl/build_id.mk
+++ b/evk_6sl/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-rc1
-export BUILD_NUMBER=20131029
+export BUILD_ID=1.0.0-rc2
+export BUILD_NUMBER=20131108
diff --git a/hdmidongle_6dq/build_id.mk b/hdmidongle_6dq/build_id.mk
index f850e0e..fe2c67b 100644
--- a/hdmidongle_6dq/build_id.mk
+++ b/hdmidongle_6dq/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-rc1
-export BUILD_NUMBER=20131029
+export BUILD_ID=1.0.0-rc2
+export BUILD_NUMBER=20131108
diff --git a/imx5x/build_id.mk b/imx5x/build_id.mk
index f850e0e..fe2c67b 100644
--- a/imx5x/build_id.mk
+++ b/imx5x/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-rc1
-export BUILD_NUMBER=20131029
+export BUILD_ID=1.0.0-rc2
+export BUILD_NUMBER=20131108
diff --git a/sabreauto_6q/build_id.mk b/sabreauto_6q/build_id.mk
index f850e0e..fe2c67b 100644
--- a/sabreauto_6q/build_id.mk
+++ b/sabreauto_6q/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-rc1
-export BUILD_NUMBER=20131029
+export BUILD_ID=1.0.0-rc2
+export BUILD_NUMBER=20131108
diff --git a/sabresd_6dq/build_id.mk b/sabresd_6dq/build_id.mk
index f850e0e..fe2c67b 100644
--- a/sabresd_6dq/build_id.mk
+++ b/sabresd_6dq/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-rc1
-export BUILD_NUMBER=20131029
+export BUILD_ID=1.0.0-rc2
+export BUILD_NUMBER=20131108
-- 
1.8.0

